pkgs/development/compilers/go/1.14.nix

```
@@ -11,8 +11,16 @@ let
 
   inherit (lib) optionals optionalString;
 
-  go_bootstrap = callPackage ./bootstrap.nix {
-    inherit Security;
+  go_bootstrap = callPackage ./binary.nix {
+    version = "1.13.15"; # Using prior version, just like Rust.
```

The `rust` part of the comment can be removed.
```suggestion
    version = "1.13.15"; # Using prior version.
```